game/tests.py

```
@@ -4044,3 +4044,41 @@ def test_unsafe_next_and_safe_referer_falls_back_to_referer(self):
     def test_no_next_and_no_referer_fallback(self):
         response = self.client.post(self.url)
         self.assertRedirects(response, reverse('forum'), fetch_redirect_response=False)
+
+
+from multiprocessing.connection import Connection
+
+class EnginePersistentTimeoutTest(SimpleTestCase):
+    @mock.patch("multiprocessing.connection.Client")
+    @mock.patch("game.engine.os.path.exists", return_value=True)
+    @mock.patch("builtins.open", new_callable=mock.mock_open, read_data="1234")
+    def test_normal_response(self, mock_open, mock_exists, mock_client):
```

Ruff (0.15.21)

[warning] Unused method argument: `mock_open`

(ARG002)